### ISSUE TITLE
[MO 2.0.4a; feat. req.] Include XML files in text or ini file section.

### ISSUE THREAD

**Issue by Freso** [TanninOne/modorganizer#585]
Some mods use XML files for configuration. It would be neat if the XML files would show up under either the "Textfiles" or "INI-Files" tab so they could be edited directly in MO.

**Comment by p0xus** [comment]
If the XMLs are in the Data folder couldn't you just navigate to where it is in MO and right click -> Open/Execute?

**Comment by Freso** [comment]
Yes, just like you can with other text files including the .ini's. :)

**Comment by p0xus** [comment]
Are we both talking about the Data tab on the right between Plugins and Saves?

**Comment by Freso** [comment]
I'm talking about the "Information" dialogue you can open by right- or double-clicking a mod in the list of mods (not the plugins in the "Plugins" tab).

**Comment by p0xus** [comment]
Ohhhh. Yeah forgot about that one. Sleep is for the weak.
